bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/DrugCoverageStatus.java

```diff
@@ -0,0 +1,55 @@
+package gov.hhs.cms.bluebutton.datapipeline.rif.model;
+
+/**
+ * Enumerate the possibly values for Drug Coverage Status for Part D
+ */
+public enum DrugCoverageStatus {
+	COVERED('C', "Covered by Part D"), SUPPLEMENTAL('E',
+			"Covered by enhanced coverage for supplemental drugs"), OVER_THE_COUNTER('O',
+					"Covered by enhanced coverage for over-the-counter Drugs");
+
+	private final Character code;
+	private final String description;
```

Comment:
Is the `description` field used by any code, or should it just be a JavaDoc comment on the constant?